recipes/unidist/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set name = "unidist" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b2cc602604f75ed6a9323cd4cd0585a1c806d3465f50ecb7ff31772b80f2edb
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - unidist = unidist.cli__main__:main
```

Comment:
This should be added to the output where you are actually building the package and a test should be added.

Something like `unidist --help`.

Reply:
Ok, but you only need to add it to `unidist` as that's the only place you're building it.

Reply:
fixed